packages/web/src/common/utils/data-check-util.ts

```diff
@@ -1,29 +1,201 @@
-export const getType = (target: any) => {
-  return Object.prototype.toString.call(target).slice(8, -1);
+/**
+ * Type name constants for ECMAScript built-in types
+ */
+const TYPE_NAMES = {
```

Comment:
This would be ok to simply declaring it as a string literal type. However, I defined it as a `Record` type because there might be cases where runtime lookups are needed.